packages/gonx/src/generators/go-blueprint/go-blueprint.ts

```diff
@@ -133,6 +149,20 @@ export async function goBlueprintGenerator(
     if (existsSync(filesDir)) {
       rmSync(filesDir, { recursive: true, force: true });
     }
+
+    // Remove unnecessary files generated by go-blueprint
+    const unnecessaryFiles = [
```

Comment:
suggestion: let's call them unsupportedFile, since they might have some usefulness but we don't support them for now.